apps/core/scenes/search/Search.js

```diff
@@ -14,74 +14,39 @@ import {
   Routes,
   type Navigation,
 } from '@kiwicom/margarita-navigation';
-import * as DateFNS from 'date-fns';
 
+import {
+  withSearchContext,
+  type SearchContextState,
+  type TripTypes,
+  type ModalTypes,
+} from './SearchContext';
 import Placepickers from './Placepickers';
-import Datepickers from './Datepickers';
+import Datepickers, { formatDate } from './Datepickers';
```

Comment:
It's a smell for me that we have to import `formatDate` from `Datepickers`. I believe it should be extracted in a new file.